test/e2e/utils/env_utils.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2eutils
+
+import (
+	"os"
+	"strings"
+
+	"github.com/onsi/ginkgo"
+	"sigs.k8s.io/azuredisk-csi-driver/test/e2e/driver"
+)
+
+const (
+	KubeconfigEnvVar = "KUBECONFIG"
+	ReportDirEnvVar  = "ARTIFACTS"
+	DefaultReportDir = "/workspace/_artifacts"
+
+	testMigrationEnvVar     = "TEST_MIGRATION"
+	testWindowsEnvVar       = "TEST_WINDOWS"
+	cloudNameEnvVar         = "AZURE_CLOUD_NAME"
+	inTreeStorageClass      = "kubernetes.io/azure-disk"
+	buildV2Driver           = "BUILD_V2"
+	useOnlyDefaultScheduler = "USE_ONLY_DEFAULT_SCHEDULER"
+	scaleTest               = "SCALE_TEST"
+)
+
+var (
+	IsUsingInTreeVolumePlugin   = os.Getenv(driver.AzureDriverNameVar) == inTreeStorageClass
+	IsTestingMigration          = os.Getenv(testMigrationEnvVar) != ""
+	IsWindowsCluster            = os.Getenv(testWindowsEnvVar) != ""
+	IsUsingCSIDriverV2          = os.Getenv(buildV2Driver) != ""
+	IsUsingOnlyDefaultScheduler = os.Getenv(useOnlyDefaultScheduler) != ""
+	IsAzureStackCloud           = strings.EqualFold(os.Getenv(cloudNameEnvVar), "AZURESTACKCLOUD")
+	// IsRunningScaleTest env variable to run scale test
+	IsRunningScaleTest = os.Getenv(scaleTest) != ""
```

Comment:
I don't see `IsRunningScaleTest` being used anywhere. Is it (and its corresponding environment variable) really needed?

Reply:
Removed